package.json

```diff
@@ -20,6 +20,7 @@
 		"globby": "^11.0.0",
 		"html-escaper": "^3.0.0",
 		"mustache": "^4.0.1",
+		"node-stream-zip": "^1.11.3",
```

Comment:
We will need a 3pp for this.

Reply:
https://gus.lightning.force.com/lightning/r/ADM_Third_Party_Software__c/a0qB0000000KpO1IAK/view